### PR TITLE
Changed import students to use username/password and allow existing users

### PR DESCRIPTION
**Pull request by jpslav**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -46,7 +46,7 @@ gem 'openstax_utilities', '~> 4.2.0'
 gem 'whenever'
 
 # OpenStax Accounts integration
-gem 'openstax_accounts', '~> 5.2.0'
+gem 'openstax_accounts', git: 'https://github.com/openstax/accounts-rails.git', ref: 'c903f1ed5a1297424f17040bf9cfa4efddf3cd0e'
```
Comment (karenc) [inline Gemfile]: Is the new version of openstax_accounts released now?

**Comment by karenc** [comment]
Looks good but wait until the next version of openstax_accounts gem is released?

File: Gemfile (hunk 1 above)
Comment (jpslav) [inline Gemfile]: Not yet, PR is in and @Dantemss needs to review and release the new version

**Comment by jpslav** [comment]
yes - will wait on accounts-rails release

**Pull request merged**